spec.md

```diff
@@ -136,6 +136,9 @@ a stand-alone event format and stored in the message body.
 A "binary-mode message" is one where the event data is stored in the message
 body, and event attributes are stored as part of message meta-data.
 
+While protocol bindings typically use "binary-mode messages" directly on the
```

Comment:
Not sure if this note belongs in the spec. It seems more like a usage/user guide detail.